Dockerfile

```diff
@@ -18,7 +18,7 @@ VOLUME /usr/share/tomcat7/logs
 RUN wget http://jdbc.postgresql.org/download/postgresql-9.3-1102.jdbc4.jar -P /var/lib/tomcat7/webapps/xwiki/WEB-INF/lib/
 
 #Download WAR from xwiki
-RUN \curl -o xwikiDownloadPage.html http://download.forge.ow2.org/xwiki/
+RUN \curl -o xwikiDownloadPage.html http://download.forge.ow2.org/xwiki/?C=M;O=A
```

Comment:
I had to wrap the second arg in quotes to get this to work.

`RUN \curl -o xwikiDownloadPage.html "http://download.forge.ow2.org/xwiki/?C=M;O=A"`